content/sensu-go/6.2/api/events.md

```diff
@@ -30,7 +30,8 @@ HTTP/1.1 200 OK
 [
   {
     "timestamp": 1542667666,
-    "id": "caaf2c38-2afb-4f96-89b3-8ca5c3e6f449",
+    "event_id": "caaf2c38-2afb-4f96-89b3-8ca5c3e6f449",
```

Comment:
The change in https://github.com/sensu/sensu-go/pull/4111 is limited only to logging context. `id` key should stay the same in API responses. I've verified this in QA:

```
$ curl -s https://caviar.tf.sensu.io:8080/api/core/v2/namespaces/default/events -H "Authorization: Key $SENSU_API_KEY" | jq '.[0]|keys'   
[
  "check",
  "entity",
  "id",
  "metadata",
  "sequence",
  "timestamp"
]
```

Reply:
Got it, thank you

Reply:
We are using event_id in the event references back to 5.20 -- I'm guessing these also need to be updated to id